src/utils.js

```diff
@@ -108,18 +109,61 @@ export default {
 	setGitConfig( name, value ) {
 		return this.exec( `${ GIT_CONFIG_COMMAND } ${ name } ${ value.trim() }` );
 	},
-	getGitConfigs() {
+	getGitConfigs( options ) {
 		return Promise.all( [
 			this.getGitConfig( "tag-release.username" ),
 			this.getGitConfig( "tag-release.token" )
-		] );
+		] )
+			.then( config => {
```

Comment:
Another way to code this with ES6 could be...

```
.then( ( [ username, token ] ) => {
  return Object.assign( options, { username, token } );
} )
```
...it destructures the array into named parts and then creates a new object and overlays that on top of the existing options.